packages/engine/Source/Core/PolygonGeometry.js

```diff
@@ -1078,6 +1080,195 @@ PolygonGeometry.unpack = function (array, startingIndex, result) {
   return result;
 };
 
+const scratchPolar = new Stereographic();
+const scratchPolarPrevious = new Stereographic();
+const scratchCartesian0 = new Cartesian2();
+const scratchCartesian1 = new Cartesian2();
+const scratchPolarClosest = new Stereographic();
+function expandRectangle(
+  position,
+  lastPosition,
+  ellipsoid,
+  arcType,
+  polygon,
+  result
+) {
+  const positionPolar = Stereographic.fromCartesian(position, scratchPolar);
+  const positionPolarPrevious = Stereographic.fromCartesian(
```

Comment:
This will project each point in the polygon twice, which looks like a performance hit. 
Perhaps this is out of scope, but could we handle the projection in the calling function and pass stereographic positions to `expandRectangle`? Then `positionPolar` could be stored (in `computeRectangleFromPositions`) for use as `positionPolarPrevious` in the next call.